examples/examples.scss

```diff
@@ -0,0 +1,287 @@
+@import "../../src/main.scss";
```

Comment:
Is scss-lint a tool that react-vis uses? I can't seem to find the dependency or npm script for adding sass linting, so I assume that it requires some kind of global installation to run. I can PR for [stylelint](https://github.com/stylelint/stylelint) later. Would that be welcome?